components/formFields.js

```diff
@@ -5,6 +5,10 @@ import DropDownPicker from 'react-native-dropdown-picker';
 import { styles } from './formFields.styles';
 import { TextSourceSans } from './TextSourceSans';
 
+// Setting list mode to scrollview globally
+// FIXME: to be checked if we want this
+DropDownPicker.setListMode('SCROLLVIEW');
```

Comment:
why do we want to check this?

Reply:
Because its set globally, meaning all DropDownPicker will be affected. Maybe what we want anyway?

Reply:
I guess we'll know more when we know what else we will build for the app, I think for now it's fine and we can keep the fixme in there to not forget about it